migrations/authgear/20240826112239-add_ldap_last_login_username.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up
+ALTER TABLE _auth_identity_ldap ADD COLUMN last_login_username TEXT;
+UPDATE _auth_identity_ldap
+SET last_login_username = '';
+ALTER TABLE _auth_identity_ldap ALTER COLUMN last_login_username SET NOT NULL;
```

Comment:
Should be okay to do this? Since this feature is still not publicly launched

Reply:
We can only add nullable column. Please change your code to gracefully handle nulls.

Reply:
Sure